app/presenters/tree_node_builder.rb

```diff
@@ -171,7 +172,8 @@ def hash_node
       :icon  => "#{object[:image] || text}.png",
       :title => ERB::Util.html_escape(text),
     }
-    @node[:expand] = true if options[:open_all] # Start with all nodes open
+    # Start with all nodes open unless expand is explicitly set to false
+    @node[:expand] = true if options[:open_all] && options[:expand] != false
```

Comment:
@chessbyte i am checking for options != false explicitly here because options[:expand] could be true or in some cases if tree_init_options didn't set it, it can be nil. I want to set @node[:expand] to be true when options[:expand] is set to true or is nil.